build-tools/libzip-windows/libzip-windows.mdproj

```diff
@@ -26,12 +26,21 @@
   <Import Project="libzip-windows.targets" />
   <Import Project="..\libzip\libzip.targets" />
   <ItemGroup>
-    <ProjectReference Include="..\android-toolchain\android-toolchain.mdproj">
+    <ProjectReference Include="..\android-toolchain\android-toolchain.mdproj" Condition=" '$(HostOS)' != 'Windows' ">
       <Project>{8FF78EB6-6FC8-46A7-8A15-EBBA9045C5FA}</Project>
       <Name>android-toolchain</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-  <Target Name="Build" DependsOnTargets="$(BuildDependsOnLocal)" />
+  <Target Name="Build" DependsOnTargets="$(BuildDependsOnLocal)" Condition=" '$(HostOS)' != 'Windows' " />
+  <Target Name="Build"
```

Comment:
This target should instead be moved into `libzip-windows.targets`.

The problem is that Xamarin Studio/Visual Studio for Mac has a nasty habit of "rewriting" all `.csproj`/`.mdproj` files referenced by the `.sln` whenever adding or removing projects, and that "rewrite" clobbers all whitespace. (Thus, the nice one-attribute-per-line format becomes one line.)